pages/index.page.js

```diff
@@ -23,9 +27,10 @@ export default function Home() {
       </Head>
 
       <main>
-        <Background imagePath="/background.png">
+        <Background imagePath={backgroundImagePath}>
```

Comment:
Should we add the same css transition effect as for the sliders? 
Currently, the background changes in the blink of an eye while the survey slide has a smooth fade-in/out effect :) 
It could be nice to use the exact same effect in both transitions :)

Reply:
I've checked at the preview today: it is working well when the image is already loaded, but still has a "immediate display" when the image finished to load after the transition happened. 
IMO it is good enough like this! ;-)